sharing_portal/templates/sharing_portal/detail.html

```diff
@@ -97,6 +97,25 @@ <h4>Digital Object Identifier (DOI)</h4>
       </p>
     {% endif %}
 
+    {% if http_content %}
+    <a class="artifactLaunchButton btn btn-primary" href="{{ download_url }}" target="_blank">Download Archive</a>
```

Comment:
I'm wondering if this would look less busy with `btn-small` (`btn-sm`?) and perhaps losing the `-primary` modifier so it's not green.

Reply:
`btn-sm` doesn't change anything. Everything in the sidebar is 100% width, so I'm not sure how this would look. Without `-primary` , there is no background color, and same with `-secondary`.